deploy/bundle/windows-machine-config-operator/manifests/windows-machine-config-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,181 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: windows-machine-config-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    alm-examples: >-
+      [{"apiVersion":"wmc.openshift.io/v1alpha1","kind":"WindowsMachineConfig","metadata":{"name":"example-windowsmachineconfig"},"spec":{"size":3}}]
+    categories: OpenShift Optional
+    certified: 'false'
+    createdAt: ''
+    description: >-
+      This operator allows for the management of Windows nodes in an OpenShift
+      cluster.
+    containerImage: ''
+    support: ''
+    capabilities: Basic Install
+    repository: ''
+spec:
+  displayName: Windows Machine Config
+  description: |
+    This operator allows for the management of Windows nodes in an OpenShift
+    cluster.
+  maturity: alpha
+  version: 0.0.1
+  replaces: ''
+  skips: []
+  minKubeVersion: ''
+  keywords: []
+  maintainers:
+    - name: ''
+      email: ''
+  provider:
+    name: Red Hat
+  labels: {}
+  selector:
+    matchLabels: {}
+  links:
+    - name: Repository
+      url: 'https://github.com/openshift/windows-machine-config-operator'
+#  icon:
+#    - base64data: ''
+#      mediatype: ''
+  customresourcedefinitions:
+    owned:
+      - name: windowsmachineconfigs.wmc.openshift.io
+        displayName: WindowsMachineConfig
+        kind: WindowsMachineConfig
+        version: v1alpha1
+        description: Windows Machine Config
+        resources:
+          - version: v1
+            kind: Deployment
+          - version: v1
+            kind: Service
+          - version: v1
+            kind: ReplicaSet
+          - version: v1
+            kind: Pod
+        specDescriptors: []
+        statusDescriptors: []
+    required: []
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+        - serviceAccountName: windows-machine-config-operator
```

Comment:
I thought this could go into a separate SA yaml and referenced here. Please confirm if this is possible with the OLM team.